generated/nidaqmx/constants.py

```diff
@@ -715,8 +727,8 @@ class Signal(Enum):
     AI_HOLD_CMPLT_EVENT = 12493
     COUNTER_OUTPUT_EVENT = 12494  #: Timed Loop executes each time the Counter Output Event occurs.
     CHANGE_DETECTION_EVENT = 12511  #: Timed Loop executes each time the Change Detection Event occurs.
-    WATCHDOG_TIMER_EXPIRED_EVENT = 12512
-    SAMPLE_COMPLETE = 12530  #: Timed Loop executes each time the Sample Complete Event occurs.
+    WDT_EXPIRED_EVENT = 12512
+    SAMPLE_COMPLETE_EVENT = 12530  #: Timed Loop executes each time the Sample Complete Event occurs.
```

Comment:
This is fine

Reply:
Updated enum values as in expected constants.py